payments/base/arch/build.gradle.kts

```diff
@@ -1,11 +1,54 @@
 plugins {
-  id(GradlePluginId.ANDROID_LIBRARY)
-  id(GradlePluginId.ANDROID_MODULE)
+  id("com.android.library")
+  id("org.jetbrains.kotlin.android")
 }
 
+apply("../../versions.gradle.kts")
+
 android {
+  val compileSdkVersion = getVersionFor("compileSdkVersion", "defaultCompileSdkVersion") as Int
+  val minSdkVersion = getVersionFor("minSdkVersion", "defaultMinSdkVersion") as Int
+
   namespace = "com.appcoins.payments.arch"
+  compileSdk = compileSdkVersion
+
+  defaultConfig {
+    minSdk = minSdkVersion
+
+    consumerProguardFiles("consumer-rules.pro")
+  }
+
+  buildTypes {
+    release {
+      proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+    }
+  }
+
+  compileOptions {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
+  }
+
+  lint {
+    abortOnError = false
+  }
 }
 
+fun getVersionFor(
```

Comment:
Can we extract it to `../../versions.gradle.kts`?
Also the default values names have the same `default` prefix. Mb we can you this fact to reduce number of arguments and just add `default_` prefix to the `versionName` value name to get dependency? 🤔